Murcielago/Murcielago/UserInterface/Views/PatternViews/SpiralPatternView.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  SpiralPatternView.swift
+//  Murcielago
+//
+//  Created by Sam Franusic on 9/13/24.
+//
+
+import SwiftUI
+
+struct SpiralPatternView: View {
+    @StateObject var outerModel = SpiralViewModel(emitters: 9)
+
+    var body: some View {
+        GeometryReader { reader in
+            ZStack {
+                RoundedRectangle(cornerRadius: 15.0)
+                    .aspectRatio(contentMode: .fit)
+                    .foregroundColor(.clear)
+                ForEach(0..<outerModel.nodes.count, id: \.self) { index in
+                    LevelView(degrees: outerModel.getAngleForNode(index))
+                        .frame(width: reader.size.width * 0.15)
+                        .offset(y: -reader.size.height * 0.4)
+                        .rotationEffect(
+                            .degrees(
+                                outerModel.getAngleForNode(index)
+                            )
+                        )
+                        .opacity(outerModel.nodes[index] == true ? 1 : 0)
```

Comment:
Maybe simplify to:

`.opacity(outerModel.nodes[index] == true ? 1 : 0)`

Reply:
Great catch. Updated